File: 프로그래머스/lv2/12914. 멀리 뛰기/멀리 뛰기.py
def solution(n):
    k = 1234567
    answer = [0] * (n)
    
    for i in range(n):
        if i == 0:
            answer[i] = 1
        elif i == 1:
            answer[i] = 2
        else:
            answer[i] = answer[i-1] + answer[i-2]
            
        
    
    return answer[n - 1] % k

File: 프로그래머스/2/12914. 멀리 뛰기/멀리 뛰기.js
function solution(n) {
    const answer = new Array(n + 1).fill(0)
    answer[1] = 1
    answer[2] = 2
    
    for (let i = 3; i <= n; i++) {
        answer[i] = (answer[i - 1] + answer[i - 2]) % 1234567
    }
    
    return answer[n];
}